tests/test_transformer_manager.py

```diff
@@ -79,10 +79,14 @@ def test_launch_transformer_jobs(self, mocker, mock_rabbit_adaptor):
         mocker.patch.object(kubernetes.client, 'AutoscalingV1Api', return_value=mock_autoscaling)
 
         transformer = TransformerManager('external-kubernetes')
+        cfg = {
+            'TRANSFORMER_CPU_LIMIT': 4,
```

Comment:
It would be good if a test verified `TRANSFORMER_MIN_REPLICAS` being explicitly applied in the HPA Spec